.github/workflows/macos.yml

```diff
@@ -0,0 +1,119 @@
+---
+name: macOS
+
+on:
+  push: {}
+
+jobs:
+  build:
+    name: py-${{ matrix.py.ver }} libgit2-${{ matrix.libgit2}}
+    runs-on: macos-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        libgit2: ['maint/v0.99']
+        py:
+          - ver: '3.6'
+            release: '3.6.8'  # last Python.org binary release
```

Comment:
FTR here's a way to select the concrete "patch" version intelligently: https://github.com/amol-/dukpy/blob/54699a3/.travis.yml#L56-L109

Reply:
Thanks - I need to select the last version available as a binary release from Python.org, not the latest patch version though. I don’t understand why they’re not built automatically, but they’re not.

Reply:
Ah, re-reading, it does that, yes! Not sure it’s a big deal for us atm - we’re not typically using cutting edge CPython features

Reply:
Yep. Sometimes they release 3.6.8 for Linux, but stop at 3.6.6 for macOS. so this is something that my script accounts for. But yes, it shouldn't matter much. OTOH when you do exit-checks after the build but before publishing the artifacts, it may make sense to run tests on the recent version.

*/me needs to find time and contribute that to cimultibuild...*